tableprinter/tableprinter_test.go

```diff
@@ -0,0 +1,163 @@
+package tableprinter
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/fluidkeys/fluidkeys/colour"
+)
+
+func TestAssembleTable(t *testing.T) {
+	tp := TablePrinter{
```

Comment:
Cool amount of testing!

I was a bit surprised to see this test direct setting the struct's data, rather than calling, `PushRow` as an outside consumer would be forced to